test/tests/utils/perf_graph_live_print/tests

```diff
@@ -1,11 +1,66 @@
 [Tests]
-  [./test]
+  [time_test]
     type = RunApp
     input = perf_graph_live_print.i
     expect_out = "Testing Slowness"
 
     issues = '#15444'
     design = 'utils/PerfGraph.md'
-    requirement = 'The system shall allow for timing sections of code and having automated print-outs when they take too long or use too much memory'
-  [../]
+    requirement = 'The system shall allow for timing sections of code and having automated print-outs when they take too long'
+  []
+
+  [nested_sections]
+    type = RunApp
+    input = perf_graph_live_print.i
+    cli_args = 'Problem/nest_inside_section=true'
+    # Checks the size of the printout as well, 62 characters after the text then a bracket
+    expect_out = "Testing Nested Slowness( |\.){62}\["
+
+    issues = '#15444'
+    design = 'utils/PerfGraphLivePrint.md'
+    requirement = 'The system shall allow for nested timing sections of code and having automated print-outs when they take too long'
+  []
+  [section_interrupted_by_regular_print]
+    # this test does not address the warnings 'direct to Moose::out' issue #23750
+    type = RunApp
+    input = perf_graph_live_print.i
+    cli_args = 'Problem/print_during_section=true'
+    expect_out = "Timed section just started, printing something though\n(Currently|Still) Executing"
+
+    issues = '#23746'
+    design = 'utils/PerfGraphLivePrint.md'
+    requirement = 'The system shall be able to print the automatic print-out of a timed section when an external print happened'
```

Comment:
This requirement isn't very clear. We shall be able to print a print-out?

Reply:
reworded